lib/terraforming/cli.rb

```diff
@@ -5,6 +5,7 @@ class CLI < Thor
     class_option :tfstate, type: :boolean, desc: "Generate tfstate"
     class_option :profile, type: :string, desc: "AWS credentials profile"
     class_option :region, type: :string, desc: "AWS region"
+    class_option :"aws-use-bundled-cert", type: :boolean , desc: "Use the bundled CA certificate from AWS SDK"
```

Comment:
__[rubocop]__
* Space found before comma.
* Line is too long. [109/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)